docs/requirements/software-requirements.md

```diff
@@ -0,0 +1,234 @@
+
+1. The use of a publicly accessible VCS platforms such as Github, Gitlab, Bitbucket or SourceHut is *REQUIRED*. This may be self-hosted by an institution rather than outsourcing it to a third party.
+2. CLARIAH is registered as an [organization](https://github.com/CLARIAH) on github, any CLARIAH software project *MAY* be hosted there.
+3. Git is *RECOMMENDED*
+4. All content, including documentation, *SHOULD* be stored alongside the code in the VCS.
+5. The source code repository *SHOULD* be publically accessible from day 1 of development; it is *MUST NOT* be reduced to a release/file-transfer mechanism for completed products. Exceptions are when things are under embargo or privacy-sensitive.
+
+### 2. A README file *MUST* be provided in the root directory of the VCS
+
+1. This should be a readable plain-text files. Markup formats such as markdown (``README.md``, or alternatively ReStructuredText, ``README.rst``) are *recommended*. Other formats such as LaTeX, HTML, Word, etc *MUST NOT* be used for the README.
+2. The README *MUST* have a clear description of what the software does (what problems it solves) and whom it is intended for.
+3. The README *MUST* make the current status of the software clear, is it ready for production, experimental, or a proof-of-concept? It *MUST* also make clear whether the software is actively maintained or not. The use of the [repostatus](https://www.repostatus.org/) vocabulary is *RECOMENDED* to this end. A simple repostatus badge suffices.
+4. The README *MUST* make clear who a) wrote the software and have a contact link, b) acknowledge the funders
+https://github.com/IdentityPython/SATOSA5. The README *MUST* provide (or link to) installation instructions.
+6. The README *MUST* provide (or link to) usage instructions for a quick start, explaining how the first task can be performed with the system.
+7. The README *should* make explicit any software and minimal hardware requirements the software relies on. This includes making explicit the software's main dependencies, target OS, and minimally required resources.
+8. The README *MUST* make clear (link to) a public support channel (e.g. an issue tracker, mailing list) where people can submit bug reports and feature requests.
+
+### 3. The software *MUST* be distributed as open source under an OSI approved license
+
+The full license *MUST* be stored in a ``LICENSE`` (``LICENSE.md``) file in the root of the VCS, the same format
+recommendations and restrictions apply as to the ``README`` file.
+
+Developers *MUST* ensure the license is not in conflict with the licenses of the dependencies they use.
+
+* See the list of [OSI-approved licenses](https://opensource.org/licenses)
+
+### 4. The software *MUST* be released periodically with clear version numbers
+
+Periodic releases of the software should be released with a clear version number.
+
+1. The use of [semantic versioning](https://semver.org/) is *strongly recommended*.
```

Comment:
‘Strongly recommended’ is not a RFC 2119 keyword.

Reply:
Right, I'll change it to "RECOMMENDED"